### ISSUE TITLE
Regression Testing MSL 3.2.2-beta.3

### ISSUE THREAD

**Issue by modelica-trac-importer** [modelica/Modelica#1949]
**Reported by leo&#x2e;gall on 3 Mar 2016 15:12 UTC**
This ticket collects result deviations of MSL 3.2.2-beta.3.
Library officers please inspect.

Reference:
- MSL v3.2.1+build.4
- Dymola 2014 FD01

To be tested:
- MSL v3.2.2+build.0-beta.3
- Dymola 2017 Beta 1

Methods for regression testing: Own mos-scripting + MA compare.exe (pre-release version 2.0.0.1) for comparison.

Number of test cases (compared to v3.2.2-beta.2):
-	Modelica: 366 (unchanged)
-	ModelicaTest: 518 compared to 543 (6 added due to new stop time annotations, 31 moved to ModelicaTestOverdetermined.mo)
For new test cases, results have been generated, which can be used as a baseline for comparing to other tools.

### Blocks

ModelicaTest.Blocks.Mean
- [compare report](http://www.ltx.de/download/MA/Compare_MSL_v3.2.2-beta.3/ModelicaTest/Mean_report.html)
- already explained for beta.2 in ticket:1924#comment:4

### Electrical.Machines

Modelica.Electrical.Analog.Examples.AmplifierWithOpAmpDetailed
- [compare report](http://www.ltx.de/download/MA/Compare_MSL_v3.2.2-beta.3/Modelica/AmplifierWithOpAmpDetailed_report.html)
- already explained for beta.2 in ticket:1924#comment:6

### Mechanics

Modelica.Mechanics.MultiBody.Examples.Systems.RobotR3.fullRobot
- compare.exe works now, thanks to fixed pre-release-version 2.0.0.1.
- [compare report](http://www.ltx.de/download/MA/Compare_MSL_v3.2.2-beta.3/Modelica/fullRobot_report.html)
- already explained for beta.2 in ticket:1924#comment:9

Modelica.Mechanics.Rotational.Examples.HeatLosses
- compare.exe works now, thanks to fixed pre-release-version 2.0.0.1.
- [compare report](http://www.ltx.de/download/MA/Compare_MSL_v3.2.2-beta.3/Modelica/2016-03-03T15-03-07-HeatLosses_report.html)
- already explained for beta.2 in ticket:1924#comment:7

Modelica.MultiBody.Joints.JointUSP2
- [compare report](http://www.ltx.de/download/MA/Compare_MSL_v3.2.2-beta.3/ModelicaTest/JointUSP2_report.html)
- already explained for beta.2 in ticket:1924#comment:10

ModelicaTest.MultiBody.Forces.SpringDamperSeries
- No stop time annotation. Please add experiment(StopTime=1.0)

ModelicaTest.MultiBody.Joints.CheckConstaintTorqueUniversalJoint
- No stop time annotation. Please add experiment(StopTime=1.0)

### Fluid

Modelica.Fluid.Examples.Tanks.TanksWithOverflow
- New deviation in beta.3
- medium model has been changed in 1aca4c08ec58114bc822448f4c04dd72a406f921, therefore deviation was probably expected?
- [compare report](http://www.ltx.de/download/MA/Compare_MSL_v3.2.2-beta.3/Modelica/TanksWithOverflow_report.html)

ModelicaTest.Fluid.Dissipation.Verifications.PressureLoss.Bend.dp_edgedOverall_DPMFLOW
- No stop time annotation. Please add experiment(StopTime=1.0)

### Access to full comparison reports

Access comparison reports of all test cases (HTML generated by MA/ITI compare.exe v2.0.0.1):
[Modelica](http://www.ltx.de/download/MA/Compare_MSL_v3.2.2-beta.3/Modelica/2016-03-03-index.html) 
[ModelicaTest](http://www.ltx.de/download/MA/Compare_MSL_v3.2.2-beta.3/ModelicaTest/2016-03-03-index.html) 

Basic documentation of the testing process is available here:
https://trac.modelica.org/Modelica/wiki/HowTo/MAP-LIB/ModelicaStandardLibrary/RegressionTesting

### Access to result files

MA file server (including mat files):
[sftp://modelica.org:20022/files/RegressionTesting/ReferenceResults/MSL/tags/v3.2.2%2Bbuild.0-beta.3/]
Attention: upload is now started and will take some hours. I'll comment this ticket a soon as upload is finished.

Readme for accessing file server:
https://svn.modelica.org/projects/RegressionTesting/AccessInfo/read-access/README.html

SVN (csv files, r9071)
https://svn.modelica.org/projects/RegressionTesting/ReferenceResults/MSL/trunk
Attention: This does not yet contain results from beta.3. Should be updated later, based on the released version.

If you have any questions replicating the test results, please don't hesitate to contact [mailto:leo.gall@ltx.de me]!

----

Migrated-From: https://trac.modelica.org/Modelica/ticket/1949

**Comment by modelica-trac-importer** [comment]
**Comment by otter on 3 Mar 2016 15:43 UTC**
StopTime annotations added in c2155916b45efed04676960b4f8babb7f391ab04:

 * ModelicaTest.MultiBody.Forces.SpringDamperSeries
 * ModelicaTest.MultiBody.Joints.CheckConstaintTorqueUniversalJoint
 * ModelicaTest.Fluid.Dissipation.Verifications.PressureLoss.Bend.dp_edgedOverall_DPMFLOW

Modelica.Fluid.Examples.Tanks.TanksWithOverflow: 
 medium model has been changed in 1aca4c08ec58114bc822448f4c04dd72a406f921 and therefore there is a deviation 

To summarize, all regressions are explained and fine.

**Comment by modelica-trac-importer** [comment]
**Comment by leo&#x2e;gall on 3 Mar 2016 16:57 UTC**
Upload of results is now finished.
​sftp://modelica.org:20022/files/RegressionTesting/ReferenceResults/MSL/tags/v3.2.2%2Bbuild.0-beta.3/

**Comment by modelica-trac-importer** [comment]
**Comment by fcasella on 3 Mar 2016 17:42 UTC**
Replying to [comment:1 otter]:
>  medium model has been changed in 1aca4c08ec58114bc822448f4c04dd72a406f921 and therefore there is a deviation 

Confirmed.

**Comment by modelica-trac-importer** [comment]
**Comment by otter on 5 Mar 2016 08:10 UTC**
Closing, since regression testing was successful
